abel/direct.py

```diff
@@ -5,6 +5,10 @@
 from __future__ import unicode_literals
 
 import numpy as np
+if hasattr(np, 'trapezoid'):  # numpy >= 2
+    trapezoid = np.trapezoid
```

Comment:
The function name change is similar to what occurred for `scipy.integrate.simp` to `simpson`. Makes more sense to use the full name.